buildSrc/src/main/java/com/google/firebase/gradle/plugins/ProjectUtils.kt

```diff
@@ -23,3 +29,45 @@ fun toBoolean(value: Any?): Boolean {
     val trimmed = value?.toString()?.trim()?.toLowerCase()
     return "true" == trimmed || "y" == trimmed || "1" == trimmed
 }
+
+/**
+ * Shorthand for [BaseExtension.getBootClasspath]
+ *
+ * This method assumes your project has the [BaseExtension] extension,
+ * as it uses [ExtensionContainer.getByType]
+ */
+val Project.bootClasspath: List<File>
+    get() = extensions.getByType<BaseExtension>().bootClasspath
```

Comment:
to be safe would it make sense to do something like the following?
```suggestion
    get() = extensions.findByType<BaseExtension>()?.bootClasspath ?: emptyList()
```

Reply:
Yeah, but it also could hide a potential bug. If the boot classpath does not exist, then theres an issue. It should always exist- even if its empty. Adding this here would mean it wouldn't throw an error when it doesn't exist- and you may not realize it. What are your thoughts?

Reply:
That's a good point. otoh this method is a partial function, i.e. its receiver is too permissive, would it make sense to change it to something like `val LibraryExtension.bootClasspath`, this will make sure that any caller has an instance of LibraryExtension, as opposed to allowing to call it on projects that can be non-android projects. also it looks like the only place that is using this has access to a LibraryExtension, wdyt?

Reply:
Actually, you may be on to something. I've changed it to just use the `LibraryExtension` directly instead.